vendor/data/codelists/IATI/2_03/activity/finance.yml

```diff
@@ -1,241 +0,0 @@
----
```

Comment:
I think we probably want to keep this file. It's useful for reference, even if we only use one code.

Reply:
I was trying to avoid having unnecessary files. There is another [yaml file](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/config/locales/codelists/2_03/iati.en.yml) in the locales for all IATI codes, including the finance ones. We could use those ones for reference if needed.

Reply:
After a conversation with @mec he explained to me the differences between vendor files and locales in terms of development, which I didn't know. Now I think it makes more sense to leave the vendor file on the code and delete the finance references from the locales. I'll do it that way